python/pyspark/streaming/flume.py

```diff
@@ -111,13 +111,9 @@ def func(event):
     @staticmethod
     def _get_helper(sc):
         try:
-            helperClass = sc._jvm.java.lang.Thread.currentThread().getContextClassLoader() \
-                .loadClass("org.apache.spark.streaming.flume.FlumeUtilsPythonHelper")
-            return helperClass.newInstance()
-        except Py4JJavaError as e:
-            # TODO: use --jar once it also work on driver
-            if 'ClassNotFoundException' in str(e.java_exception):
```

Comment:
Let's still keep this check. For other errors (e.g., the py4j java server is down), we should not call `_printErrorMsg` as it's confusing.